ts/features/walletV3/payment/store/__tests__/store.test.ts

```diff
@@ -21,4 +24,22 @@ describe("Test Wallet reducer", () => {
     const globalState = appReducer(undefined, applicationChangeState("active"));
     expect(globalState.features.wallet.payment).toStrictEqual(INITIAL_STATE);
   });
+
+  it("should update payment step", () => {
```

Comment:
```suggestion
  it("should correctly update payment step, also when trying to overflow the steps, it should set the steps to WALLET_PAYMENT_STEP_MAX, and in case zero is passed it should set the step to 1", () => {
```